### DISCUSSION TITLE
Account suspended and support Ignores me

### DISCUSSION THREAD

**Discussion by mferitsimsek** [community/community#127649] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

When trying to access my account, I received a message saying “Account suspended. Access to your account has been suspended due to a violation of our Terms of Service." message,
I can't think of anything except that it could have been due to several failed login attempts, and on the 3rd or 4th attempt I found the correct password and saw the message “Account suspended”. I didn't receive any warning message about what caused the suspension (if I had, I would have removed the violation before the suspension).
So of course I immediately sent a letter of objection,

Now the strange thing is that they haven't responded for 1 week, which makes me think that something is broken in the appeal system.

I would like to know if you can upgrade the appeal or do something about it; my original account is github.com/cptmfs 

My ticket number : #2817983

Thank you.

**Comment by mferitsimsek** [comment]
As an active software developer, not being able to access my Github account for so long is a serious problem for me. I have problems in my job interviews because my github account cannot be accessed.

Please help me.  @dmilburn @adnamalin

**Reply by queenofcorgis** [reply]
Hi there,

Please note, contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our Code of Conduct for more information.

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Comment by mohamad-ahmad-13** [comment]
same issue here. account got suspended for no reason at all. i haven't logged in to my account for 2 weeks and when i tried to login 3 days ago it says suspended for violating terms of agreement.
i need to collaborate with my colleagues on our graduation project and this is hindering my work, i need the account back.

Ticket ID: 2828265

**Comment by queenofcorgis** [comment, marked as answer]
Hey there!  👋 

Thanks for posting in the GitHub Community, @mferitsimsek ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
